docs/stepbystep.md

```diff
@@ -8,28 +8,28 @@ Step by Step guide
 * [Provisioning multiple devices with a Configuration](#withconfiguration)
 * [Using ACLs to secure provisioning access](#acls)
 
-## <a name="introduction"/> Introduction
+## <a name="introduction">Introduction</a>
```

Comment:
Is this change fixing some issue? Or is it just stetical?

Reply:
I mean, I'm not a fan of so ancient veresion :), but we should have a good reason to change

Reply:
Oh sure, neither do I, but yes. It was causing the whole text of the section to be part of the link (checkout http://fiware-iotagent-json.readthedocs.io/en/latest/stepbystep/index.html)

Reply:
I see

NTC